.github/generate-c3-dependabot-pr-changeset.mjs

```diff
@@ -0,0 +1,59 @@
+import { execSync } from "child_process";
+import { writeFileSync } from "fs";
+
+const diff = execSync(
+	"git diff HEAD~1 packages/create-cloudflare/src/frameworks/package.json"
+).toString();
+
+const changedPackages =
+	diff
+		.match(/-\s*".*":\s".*",?/g)
```

Comment:
This regex looks greedy and might need this change. Please confirm before accepting this suggestion.

```suggestion
		.match(/-\s*".*?":\s".*?",?/g)
```

Reply:
nice suggestion, thanks 🙂

the current version works fine since `.` doesn't include newlines, so even though greedy since we need to match the whole thing everything still work fine

But your version seems more correct anyways 🙂👍 
(I did double check it and it does work as intended: https://github.com/dario-piotrowicz/dependabot-testing/pull/20/files)